EIPS/eip-4844.md

```diff
@@ -174,6 +174,13 @@ def get_origin(tx: SignedBlobTransaction) -> Address:
     return ecrecover(tx_hash(tx), int(sig.y_parity)+27, sig.r, sig.s)
 ```
 
+The transaction hash of a signed blob transaction should be computed per [`EIP-2718`](./eip-2718.md):
```

Comment:
EIP-2718 doesn't specify how transaction hashes are calculated.  I *believe* that transaction hashes are the `keccak256` of the bytes in the transaction trie whose root is in the block header, so if the below aligns with that then it is correct.
```suggestion
The transaction hash of a signed blob transaction should be computed as:
```